packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -88,35 +85,18 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 		let text: React.ReactNode = null;
 		if ( isPremium && shouldUpgrade ) {
 			if ( isEnabled( 'signup/seller-upgrade-modal' ) ) {
-				text = createInterpolateElement(
-					sprintf(
-						/* translators: %(price)s - the price of the theme */
-						__( '%(price)s per year or <button>included in WordPress.com Premium</button>' ),
-						{
-							price: design.price,
-						}
-					),
+				text = sprintf(
+					/* translators: %(price)s - the price of the theme */
+					__( '%(price)s per year or included in WordPress.com Premium' ),
 					{
-						button: (
-							<Button
-								isLink={ true }
-								className="design-picker__button-link"
-								onClick={ ( e: any ) => {
-									e.stopPropagation();
-									onCheckout?.();
-								} }
-							/>
-						),
+						price: design.price,
 					}
 				);
 			} else {
-				text = (
-					<Button isLink={ true } className="design-picker__button-link">
-						{ 'en' === locale || hasTranslation( 'Included in WordPress.com Premium' )
-							? __( 'Included in WordPress.com Premium' )
-							: __( 'Upgrade to Premium' ) }
-					</Button>
-				);
+				text =
+					'en' === locale || hasTranslation( 'Included in WordPress.com Premium' )
+						? __( 'Included in WordPress.com Premium' )
+						: __( 'Upgrade to Premium' );
```

Comment:
I think we need to update the copy. It says “Upgrade to Premium” but I cannot do anything here. When I click on it, the preview will show and it's a little weird. Maybe something like “You need to upgrade to Premium”

Reply:
Since we've removed the check for the `signup/seller-upgrade-modal`, this copy has also been removed.